### ISSUE TITLE
Combine withLocalState and withStore

### ISSUE THREAD

**Issue by Dan-Do** [ficusjs/ficusjs#9]
I wonder if I used it wrong.
I got exception `Uncaught TypeError: can't access property "state", e is undefined`
```javascript
const app_state = createAppState(...);
const viewMain = createCustomElement('view-main', ExtensionBuilder.newInstance()
.withStore(app_state)
.withLocalState( { state() { return {view: "/"} } } )
.create({
  props: {route: {type: String, required: true, observed: false}},
  renderer,
  render() { return html`something`; }
})
);
```

**Comment by ducksoupdev** [comment]
Hi @Dan-Do, thanks for raising the issue. I managed to re-create the problem and have fixed it in release v3.15.0.

I have also created a Codepen with your example so you can see it working - https://codepen.io/ducksoupdev/pen/BawMYbO?editors=1010

**Comment by Dan-Do** [comment]
It works now. Thank you for this awesome library.
